index.html

```diff
@@ -1956,39 +1956,9 @@ <h3>Securing Verifiable Credentials</h3>
 mutually exclusive.
         </p>
         <p>
-<a>Verifiable credentials</a> and <a>verifiable presentations</a> MUST be secured by at least
-one securing method.
+<a>Verifiable credentials</a> MUST be secured by at least one securing method.
         </p>
-        <p>
-Methods of securing <a>verifiable credentials</a> or
-<a>verifiable presentations</a> that embed a proof in the data model MUST use
```

Comment:
This is data integrity proof specific, and should be a normative statement in that specification.